pkg/util/trace.go

```diff
@@ -0,0 +1,23 @@
+package util
+
+import (
+	"log"
+
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/trace"
+)
+
+// InitialiseTracing initialises trace, exporter and the sampler
+func InitialiseTracing(serviceName string, address string, probability float64) {
+	oce, err := ocagent.NewExporter(
+		ocagent.WithInsecure(),
+		ocagent.WithAddress(address),
+		ocagent.WithServiceName(serviceName))
+	if err != nil {
+		log.Fatalf("Failed to create ocagent-exporter: %v", err)
```

Comment:
I don't think we want to panic if we can't create an exporter.  Logging at Error level should be fine.